api/chalicelib/core/projects.py

```diff
@@ -94,25 +94,29 @@ def get_projects(tenant_id: int, gdpr: bool = False, recorded: bool = False):
                 r.pop("first_recorded")
                 r.pop("sessions_last_check_at")
             if len(u_values) > 0:
-                query = cur.mogrify(f"""UPDATE public.projects 
+                await cnx.execute(f"""UPDATE public.projects
```

Comment:
why did you replace mogrify by execute directly ?

Reply:
There is an explanation in the associated commit: https://github.com/openreplay/openreplay/pull/1701/commits/dc271df452a0b6ceb099334cedb09249adbf3f2a